lib/component-data/pubsub.js

```diff
@@ -4,6 +4,9 @@ import { pubProp, subProp, componentRoute, refProp, getComponentName } from '../
 import { ADD_SCHEMA } from './mutationTypes';
 import { PRELOAD_SUCCESS } from '../preloader/mutationTypes';
 import { props } from '../utils/promises';
+import Vue from 'vue';
+
+const vue = new Vue;
```

Comment:
If I'm not mistaken, I think we have to use `Vue()` in order to instantiate it
```suggestion
const vue = new Vue();
```

Reply:
JavaScript lets you omit parens when declaring a new Object when there are no arguments, but I went ahead and added them to avoid confusion.